fin_control/README.md

```diff
@@ -0,0 +1,9 @@
+# Fin control package.
+The fin control:
+- Sets the angle sent by Jaus ROS Bridge or the autopilot (throug the [command actuator multiplexer](catkin_ws/src/public/cmd_actuators_mux/README.md)).
+- Reports fins angles.
+
+## Diagnostic
```

Comment:
@patapufate I like the subtitle, but this is the only README that includes it.

Reply:
Add subtitle to the rest of nodes.